extensions/wikia/CategoryExhibition/CategoryExhibitionSection.class.php

```diff
@@ -5,6 +5,7 @@
  */
 class CategoryExhibitionSection {
 	const CACHE_VERSION = 3;
+	const EXHIBITION_LIMIT = 2000;
```

Comment:
This limit could use a word of explanation - what is the rationale behind it being 2000 and not for example 1500, as this knowledge is available neither in code nor in the referenced ticket.